Okta/OktaRefresh.swift

```diff
@@ -20,27 +20,10 @@ internal struct Refresh {
                 return reject(OktaError.noTokens)
             }
 
-            guard let refreshToken = tokens.refreshToken else {
+            guard let _ = tokens.refreshToken else {
                 return reject(OktaError.noRefreshToken)
             }
 
-            if tokens.authState.lastAuthorizationResponse == nil {
```

Comment:
This was required because the Resource Owner Password Flow doesn't communicate with the `/authorize` endpoint, therefore there is no `lastAuthorizationResponse`. In our [tests](https://github.com/okta/okta-sdk-appauth-ios/pull/93/commits/1f5b72680031065fb4f61ec9916bc0aa81b5d9f9#diff-bbd4e38e7f80647a76f0263d6f964857L66), we were mocking out the behavior of direct communication to the `/token` endpoint, instead of properly mocking `/authorize` -> `/token`.